### PR TITLE
841 image modal tags authors

### PR DESCRIPTION
**Pull request by Rowansdabomb**

### CODE REVIEW

File: dispatch/static/manager/src/js/components/modals/ImageManager/index.js [hunk 1]
```diff
@@ -43,16 +46,30 @@ class ImageManagerComponent extends React.Component {
...
         limit: prevState.limit + 25
-      }), this.props.listImages(this.props.token, R.assoc('q', this.state.q, {limit: this.state.limit, ordering: '-created_at' })))
+      }), this.props.listImages(this.props.token, this.buildQuery()))
     }
   }
 
   searchImages() {
-    this.props.listImages(this.props.token, R.assoc('q', this.state.q, DEFAULT_QUERY))
+
+    this.props.listImages(this.props.token, this.buildQuery())
```
Comment (Rowansdabomb) [inline dispatch/static/manager/src/js/components/modals/ImageManager/index.js]: remove space

File: dispatch/static/manager/src/js/components/modals/ImageManager/index.js [hunk 2]
```diff
@@ -91,8 +108,12 @@ class ImageManagerComponent extends React.Component {
     }
   }
 
-  onSearch(q) {
-    this.setState({ q: q }, this.searchImages)
+  onSearch(author, tags, q) {
+    this.setState({ 
+      author: author,
+      tags: tags === '' ? null: tags,
+      q: q
```
Comment (Rowansdabomb) [inline dispatch/static/manager/src/js/components/modals/ImageManager/index.js]: refactor

File: dispatch/static/manager/src/js/components/modals/ImageManager/index.js [hunk 3]
```diff
@@ -130,12 +150,21 @@ class ImageManagerComponent extends React.Component {
             <AnchorButton
               intent={Intent.SUCCESS}
               onClick={() => this.dropzone.open()}>Upload</AnchorButton>
+
```
Comment (JamieRL) [inline dispatch/static/manager/src/js/components/modals/ImageManager/index.js]: extra space

**Review by JamieRL — CHANGES_REQUESTED** [review]
Just one extra space. Looks good

**Review by JamieRL — APPROVED** [review]

**Pull request merged**